lib/preview-web/src/StoryRender.ts

```diff
@@ -242,12 +248,11 @@ export class StoryRender<TFramework extends AnyFramework> implements Render<TFra
   // as a method to abort them, ASAP, but this is not foolproof as we cannot control what
   // happens inside the user's code.
   cancelRender() {
-    if (this.abortController) {
-      this.abortController.abort();
-    }
+    this.abortController?.abort();
   }
 
   async teardown(options: {} = {}) {
+    this.torndown = true;
```

Comment:
can you elaborate what `this.torndown` is useful for?
Seems not super related to this PR?

Reply:
okay